apps/fishing-map/features/workspace/highlight-panel/highlight-panel.content.ts

```diff
@@ -22,28 +22,28 @@ export type HighlighPanelConfig = {
 const highlightPanelConfig: HighlighPanelConfig = {
   dataviewInstanceId: HIGHLIGHT_DATAVIEW_INSTANCE_ID,
   delayed: true,
-  imageUrl: `${PATH_BASENAME}/images/belize-vms.jpg`,
+  imageUrl: `${PATH_BASENAME}/images/viirs-match.webp`,
```

Comment:
I'm not sure safari supports .webp yet...and I saw a few people in the retreat using safari so I'd use jpg to ensure it works everywhere.